yogstation/code/game/objects/items/tools.dm

```diff
@@ -0,0 +1,128 @@
+	materials = list(MAT_METAL=150,MAT_SILVER=50,MAT_TITANIUM=25)
+	desc = "A set of jaws of life, compressed through the magic of science. It's fitted with a cutting head."
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "jaws_cutter"
+	item_state = "jawsoflife"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+
+	usesound = 'sound/items/jaws_cut.ogg'
+	force = 15
+	toolspeed = 0.7
+	tool_behaviour = TOOL_WIRECUTTER
+
+//jaws of life suicide code
+/obj/item/jawsoflife/suicide_act(mob/user)
+	switch(tool_behaviour)
+		if(TOOL_CROWBAR)
+			user.visible_message("<span class='suicide'>[user] is putting [user.p_their()] head in [src], it looks like [user.p_theyre()] trying to commit suicide!</span>")
+			playsound(loc, 'sound/items/jaws_pry.ogg', 50, 1, -1)
+			return (BRUTELOSS)
+		if(TOOL_WIRECUTTER)
+			user.visible_message("<span class='suicide'>[user] is wrapping \the [src] around [user.p_their()] neck. It looks like [user.p_theyre()] trying to rip [user.p_their()] head off!</span>")
+			playsound(loc, 'sound/items/jaws_cut.ogg', 50, 1, -1)
+			if(iscarbon(user))
+				var/mob/living/carbon/C = user
+				var/obj/item/bodypart/BP = C.get_bodypart(BODY_ZONE_HEAD)
+				if(BP)
+					BP.drop_limb()
+					playsound(loc,pick('sound/misc/desceration-01.ogg','sound/misc/desceration-02.ogg','sound/misc/desceration-01.ogg') ,50, 1, -1)
+			return (BRUTELOSS)
+
+//jaws of life changing jaw code
+/obj/item/jawsoflife/attack_self(mob/user)
+	playsound(get_turf(user), 'sound/items/change_jaws.ogg', 50, 1)
+	if (tool_behaviour == TOOL_CROWBAR)
+		desc = "A set of jaws of life, compressed through the magic of science. It's fitted with a cutting head."
+		if (iscyborg(user))
+			to_chat(user,"<span class='notice'>Your servos whirr as the prying head reconfigures into a cutting head.</span>")
+		else
+			to_chat(user, "<span class='notice'>You attach the cutting jaws to [src].</span>")
+		attack_verb = list("pinched", "nipped")
+		icon_state = "jaws_cutter"
+		hitsound = 'sound/items/jaws_cut.ogg'
+		usesound = 'sound/items/jaws_cut.ogg'
+		tool_behaviour = TOOL_WIRECUTTER
+		update_icon()
+	else if (tool_behaviour == TOOL_WIRECUTTER)
+		desc = "A set of jaws of life, compressed through the magic of science. It's fitted with a prying head."
+		if (iscyborg(user))
+			to_chat(user,"<span class='notice'>Your servos whirr as the cutting head reconfigures into a prying head.</span>")
+		else
+			to_chat(user, "<span class='notice'>You attach the pry jaws to [src].</span>")
+		attack_verb = list("attacked", "bashed", "battered", "bludgeoned", "whacked")
+		usesound = 'sound/items/jaws_pry.ogg'
+		hitsound = 'sound/items/jaws_pry.ogg'
+		tool_behaviour = TOOL_CROWBAR
+		icon_state = "jaws_pry"
+		update_icon()
+	else
+		to_chat(user,"<span class='warning'>You shouldn't be able to see this! Please contact a coder!</span>")
+		playsound(get_turf(user), 'sound/effects/adminhelp.ogg', 50, 1)
```

Comment:
```suggestion
```
Is this necessary? 
The tool can only be in corwbar or wirecutter mode unless an admin messes with the tool_behavior variable.
Even if there is some future PR where we introduce a wizard spell that makes all tools behave as wrenches we shouldn't bwoink people and tell them to contact a coder.

Reply:
I consider it a necessity. Consider it my signature coding style.

Reply:
Your signature style is to have an in-game tool literally play the admin bwoink sound and to tell people to contact a coder when its not behaving exactly how you predicted it would?

Reply:
yes it's called a bug

Reply:
In my opinion this is introducing a future "bug" so if someone makes an item that changes the tool behavior to another tool it is quite literally going to be used as a bwoink sound spam tool.

Reply:
In what world would this even be a "bug" that needs to be fixed?
If something changes the tool behavior to something else it would be a "feature". 

So some future coder is going to laugh it off and delete this anyway.

If you don't want your tool to behave this way then make it check if one of the tool behaviors exist first then return if they both don't exist.
This is "shitcode"

Reply:
No, what are you on about, it doesn't support other tool types, either he adds support for other tools, or this stays.
This should stay either way, if someone set tool behaviour to something undefined

Reply:
Then why not just check to see if the tool is in one of the supported modes then return if its not?

Why make the item literally bwoink you and tell you to contact the coder when the coder is just going to delete this part to "fix" it

Reply:
because that's shitcode. First thing uni courses teach you is to handle errors.

Reply:
We're having a miscommunication.
You're not handling errors you're giving a pointless message and sound.

Handling errors would be to check if your tools are in one of the modes then not allowing it to proceed unless it is in one of the modes you support.

Reply:
He just needs to add a return statement then.
Failing without giving a reason is BAD in this case

Reply:
Gamer is wrong.
This is out of place.
Fix it.

Not sure why you have a habit of refusing to change what is needed for me to merge and both times so far it was some trivial reason.

Reply:
Why is this out of place?
It's telling the user that their item is bugged